convert2rhel/utils.py

```diff
@@ -620,4 +627,52 @@ def set_locale():
     os.environ.update({"LC_ALL": "C", "LANG": "C"})
 
 
+def options_from_config_files(config_opts, cfg_path=None):
+    """Get the options from config file.
+
+    The order is:
+    1. if filename is set then load from it
+    2. ~/.convert2rhel.ini
+    3. /etc/convert2rhel.ini
+
+    Return config_opts filled with loaded values.
+    """
+    headers = ["subscription_manager"]  # supported sections in config file
+    config_file = configparser.ConfigParser()
+    paths = CONFIG_PATHS
+    # create dict with all supproted options, all of them set to None
+    # needed for avoiding problems with files priority
+    supproted_opts = {"password": None, "activation_key": None}
+
+    if cfg_path:
+        if not os.path.exists(cfg_path):
+            raise FileNotFoundError
+        paths.insert(0, cfg_path)  # highest priority
+
+    for path in paths:
+        if os.path.exists(path):
+            if oct(os.stat(path).st_mode)[-4:] != "0600":
+                loggerinst.critical("The %s file need to be in mode 0600" % path)
```

Comment:
* On Python2, `oct(384)` => "0600" but on Python3, it yields "0o600".
* We probably are okay with being only readable by the owner, readable and executable by the owner, or readable, writable, and executable by the owner.  So maybe it is better to check that the bits we don't want are not present:

```suggestion
            if not oct(os.stat(path).st_mode)[-4:].endswith("00"):
                loggerinst.critical("The %s file must only be accessible by the owner (0600)" % path)
```